web/server.js

```diff
@@ -129,27 +119,25 @@ module.exports = function(options) {
     return false;
   }
 
-  server.register({
-    register: require('crumb'),
+  await server.register({
+    plugin: require('crumb'),
     options: {
       restful: true,
       skip: !options.enableCSRF ? skipCSRF : undefined,
       cookieOptions: {
-        isSecure: options.secureCookies
+        isSecure: options.secureCookies,
+        isSameSite: false,
+        isHttpOnly: false
```

Comment:
also removes the HttpOnly header (default behaviour before)

Reply:
Pretty sure we don't want the CSRF crumb cookie accessible over Javascript?